.github/workflows/action.yml

```diff
@@ -73,19 +73,6 @@ jobs:
         extensions: ast, bcmath, gd
         coverage: none
 
-      #https://github.com/actions/cache/blob/master/examples.md#php---composer
```

Comment:
Why is this deleted? Be careful when deleting steps from the actions

Reply:
I made those changes to test if it is causing the inspection runs, removed now. Ready for review @olsi-qose